src/ucp/core/ucp_worker.c

```diff
@@ -2526,18 +2528,36 @@ static void ucp_worker_destroy_eps(ucp_worker_h worker,
     }
 }
 
+static void ucp_worker_eps_cleanup(ucp_worker_h worker)
+{
+    /* Cleanup hash of discarded UCT EPs prior destroying all UCP EPs to
+     * destroy UCP EPs which were already marked as closed by a user, but had
+     * discarding is in progress */
```

Comment:
Minor comment.
The sentence is difficult to understand. Guess it would be better to break the sentence into a few simple ones.

Reply:
done